### DISCUSSION TITLE
Go back to Netrw?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [LazyVim/LazyVim#1432] [Q&A] [answered]
Besides disabling `Neo-tree` what else is needed to go back to `Netrw`? And why is there no questions on this topic already, is `neo-tree` that good?

**Comment by dpetka2001** [comment]
You can already use `Netrw` just like you did. It's not disabled. Just do `:Explore` and you will get `Netrw`.

**Reply by Nikola-Milovic** [reply]
`not an editor command Explore`

**Reply by dpetka2001** [reply]
Can you show your `/lua/config/lazy.lua` contents?

**Reply by dpetka2001** [reply, marked as answer]
If [these](https://github.com/Nikola-Milovic/.dotfiles/blob/master/nvim/.config/nvim/lua/config/lazy.lua) are your dotfiles, then you have disabled `Netrw` yourself in this [commit](https://github.com/Nikola-Milovic/.dotfiles/commit/a95bdf2a4cbebfb6b70c017585358b9129c3e7e4#diff-126a92e2c135fae69cd17901ba411453ea321752cc16afbc199aa22780060704). If you re-enable it you should be able to run the command and use it as you wish.
